packages/@aws-cdk/aws-ec2/README.md

```diff
@@ -217,6 +217,42 @@ new ec2.Vpc(this, 'TheVPC', {
 provider.connections.allowFrom(ec2.Peer.ipv4('1.2.3.4/8'), ec2.Port.tcp(80));
 ```
 
+### Ip Address Management - IPAM
```

Comment:
You probably need to update other examples in here that pass the plain `cidr` property (now deprecated) as well.

Reply:
updates made for other samples in the readme using 'cidr'